.github/workflows/test.yml

```diff
@@ -50,30 +50,49 @@ jobs:
             dcpy:
               - dcpy/**
               - pyproject.toml
+              - python/**
             data-library:
               - data-library/**.py
               - data-library/**.toml
               - data-library/**.yml
+              - docker/**
+              - python/**
             build:
               - .github/workflows/test.yml
               - .github/workflows/build.yml
               - dcpy/**
               - products/template/**
+              - docker/**
+              - python/**
```

Comment:
The docker ones are a little funny in that they generally still require publishing a new image. But regardless in general figure it makes sense to test things if we're changing build env, be it in docker stuff or python req versions